hack/update-deps.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+glide update
```

Comment:
any reason not to use `--strip-vendor`?

Reply:
> any reason not to use --strip-vendor?

Unexposed dependencies should remain private to allow for separate evolution of dependent repos.